openlibrary/components/BulkSearch/components/MatchTable.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div v-if="bulkSearchState.matchedBooks.length" class="tableWrapper">
```

Comment:
Oh I meant move this up to the parent component! That way we entirely avoid creating/initializing this component when there are no matches.

```suggestion
  <div class="tableWrapper">
```